PressureSensorAdapter.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Random;
+import sensor.PressureSensor;
+public class PressureSensorAdapter implements SensorInterface{
+    private PressureSensor adaptee;
+    private double pressure;
+
+    public PressureSensorAdapter(){
+
+    }
+    @Override
+    public double readValue(){

```

Comment:
This is the adapter class. Adapter classes shouldn't have any implementation in them. 

Here's an example:
public double readValue(){
adaptee.readValue();
}
public String getReport(){
adaptee.getReport();
}

and so on.